specs/wp-manage-domains-spec.js

```diff
@@ -201,7 +201,7 @@ describe( `[${ host }] Managing Domains: (${ screenSize })`, function() {
 		} );
 	} );
 
-	describe( 'Transfer a domain to an existing site (partial) @parallel', function() {
+	describe.only( 'Transfer a domain to an existing site (partial) @parallel', function() {
```

Comment:
@southp Was this `.only` left intentionally? I've almost missed it, but I saw that test run was only [~1min long](https://circleci.com/gh/Automattic/wp-e2e-tests/25826)  :)

Reply:
Argh! Thanks for catching it. I've removed it in d3aa3ba.